terminal.go

```diff
@@ -689,16 +688,16 @@ func (g *LightningTerminal) start() error {
 		// daemons. In all other cases we want default macaroons so we
 		// can use the CLI tools to interact with loop/pool/faraday.
 		macService := g.lndInterceptorChain.MacaroonService()
-		createDefaultMacaroons = !macService.StatelessInit
+		g.cfg.statelessInitMode = macService.StatelessInit
 	}
 
 	// Both connection types are ready now, let's start our sub-servers if
 	// they should be started locally as an integrated service.
 	g.subServerMgr.StartIntegratedServers(
-		g.basicClient, g.lndClient, createDefaultMacaroons,
+		g.basicClient, g.lndClient, !g.cfg.statelessInitMode,
```

Comment:
nit: to give this boolean a bit more context for how it's being used by the subservers, should we just do `createDefaultMacaroons := !g.cfg.statelessInitMode` above this function call?